mithril-signer/src/services/aggregator_client.rs

```diff
@@ -553,6 +554,33 @@ mod tests {
         };
     }
 
+    fn mock_server_verifies_accept_encoding_header_values_unsorted(
+        server: &MockServer,
+        http_code_returned: u16,
+        body_returned: Option<String>,
```

Comment:
This is quite difficult to understand.

The `http_code_returned` and `body_returned` parameters should be removed and handled directly in the tests.
Also, I think this method could me renamed to something closer to the other test utils here, something like: `check_accept_encoding_headers`.